src/main/java/org/jabref/gui/EntryTypeViewModel.java

```diff
@@ -60,6 +61,7 @@ public EntryTypeViewModel(PreferencesService preferences,
         this.dialogService = dialogService;
         this.stateManager = stateManager;
         fetchers.addAll(WebFetchers.getIdBasedFetchers(preferences.getImportFormatPreferences()));
+        fetchers.sort(Comparator.comparing(IdBasedFetcher::getName));
```

Comment:
As I said in a comment I don't think it's useful to sort by alphatical order here. DOI fetcher should always be on top because it's the most important one

Reply:
So please revert that